File: Lab5/Lab5_C++.Task2/Row.cpp
#include "Row.h"

Row::Row(string Row) {
	_Row = Row;
}

string Row::Value() { return this->_Row; }

int Row::Length() { return this->_Row.length(); }

void Row::Remove(char ch) {
	string temp;
	for (auto& i : _Row) {
		if (i == ch)
			continue;
		temp.push_back(i);
	}
	_Row = temp;
}

File: Lab5/Lab5_CS.Task2/Row.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Lab5_CS.Task2
{
    class Row
    {
        protected string _Row;
        public string Value => _Row;
        public Row(string Row)
        {
            _Row = Row;
        }
        public virtual int Length() => _Row.Length; // returns length of string.
        public virtual void Remove(char ch) => _Row = new string(_Row.Where(c => !ch.ToString().Contains(c)).ToArray()); // Creates a new string, if it meets char to remove, it skips it.
    }
}
